stage1/reaper/reaper.sh

```diff
@@ -7,5 +7,11 @@ if [ $# -eq 1 ]; then
     app=$1
     status=$(${SYSCTL} show --property ExecMainStatus "${app}.service")
     echo "${status#*=}" > "/rkt/status/$app"
+    if [ "${status#*=}" != 0 ] ; then
+        # The command "systemctl exit $status" is available since systemd v227.
+        # On older versions, the command will fail and rkt will just exit with
+        # return code 0.
+        ${SYSCTL} exit ${status#*=}
+    fi
```

Comment:
Tested on the current `coreos` flavor:

```
root@rkt-25122456-cb0d-4dae-bd21-905c754c22ed:/# exit 5
exit
Invalid number of arguments.
```

What about using:
```
${SYSCTL} exit ${status#*=} 2>/dev/null
```

I don't think we can nor should parse the output of `systemctl --version` since we don't have `grep` and other tools in stage1...